doc/classes/BoneExpander3D.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="BoneExpander3D" inherits="SkeletonModifier3D" experimental="This SkeletonModifier3D interferes with the Skin in a slightly unsafe way. Performance and stability have not been sufficiently optimized." xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Scales the visual appearance of the specified bone without causing shear in the children.
+	</brief_description>
+	<description>
+		Scales only the visual appearance without changing the bone pose scale. This is achieved by modifying the bone pose position and skin bindings instead of the bone pose scale.
+		In other words, the only thing other modifiers can obtain from this modifier via the bone is the bone pose position.
```

Comment:
```suggestion
		Scales only the visual appearance without changing the bone pose scale. This is achieved by modifying the bone pose position and skin bindings instead of the bone pose scale. In other words, the only thing other modifiers can obtain from this modifier via the bone is the bone pose position.
```